cpp/src/reductions/minmax.cu

```diff
@@ -60,7 +60,6 @@ struct minmax_pair {
  * @param d_in input iterator
  * @param num_items number of items to reduce
  * @param binary_op binary operator used to reduce
- * @param mr Device resource used for result allocation
```

Comment:
Would it make sense to document `T`, `OutputType`, as part of this change?

Reply:
`T` template param is not needed in this struct. removed it.
added documentation for `OutputType`.